### DISCUSSION TITLE
Community Calls

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1282] [Operations]
### purpose
- introduce and welcome new members to the community
- discuss topics in/with the community

### current state
- Every Sunday at 2p EDT we have a community call in the meeting-room voice channel of our discord server. 
- process
  - Announcement of call is made in announcements channel on discord and on twitter
  - Apollo bot posts to events channel in discord where discord members can mark their attendance
  - calls are recorded and posted to loom or youtube unlisted
  - meeting notes are taken in public athens roam database
  - description of call, meeting notes, recording, and highlights are posted into meeting-notes channel
- calendar here is not updated with specific community call event details https://calendar.google.com/calendar/embed?src=researchathens%40gmail.com&ctz=America%2FLos_Angeles

### questions
- how can we make calls more inclusive?
- how can we make calls more accessible?
  - we announce calls in the discord, should advertise in monthly newsletter as well
- generally speaking, how can we improve our community calls?

### notes
- Informally, we give new individuals the opportunity to introduce themselves
- calls either jump right into the topic
- we currently record all calls and share them on loom or unlisted on our youtube channel
- for call recording and consent we should be sure to make recording known in the announcement and at the beginning of every call
- Topics in the past have included:
  - guest speakers
  - codebase walkthrough
  - unstructured/open ended
  - Q & A
  - tech demos (ex. Fluree)
  - monthly priorities
  - roadmap discussion
  - product updates
  - project updates
  - feature testing (ex. RTC)
  - company announcements

**Comment by joelhans** [comment]
I liked when Jeff created a discussion thread for community calls (#1253/#1139) and I wonder if we can't add that to the process. Let's also try to put some dates behind each step in the process. Maybe we can use the individual discussion threads for project management as well?

Here's a template we could re-use every week to coordinate this and keep on top of various tasks along the way. Let me know what I'm missing in the process so it covers all the steps you've already been taking @jsmorabito!

```
**Community call**: {DATE}
**Calendar invite**: {CALENDAR}
**Topic**: {TOPIC}
**Agenda**: 
  - Introductions from community members
  - item
  - item
  - item
**Recording link**: TBD
**Link to notes**: TBD

---

- T-4 days **Wednesday**:
  - [ ] Create a GitHub discussion thread with this template.
  - [ ] Post in the #announcements channel on [Discord](https://discord.gg/as9h8yHNfD).
  - [ ] Set up Apollo to post the event in the #events channel on [Discord](https://discord.gg/as9h8yHNfD).
  - [ ] Post to Twitter.
- T-1 day **Saturday**:
  - [ ] Post a reminder in the #announcements channel.
- T-0 **Sunday** (before call begins):
  - [ ] Post a final reminder in the #announcements channel.
  - [ ] Post a reminder to Twitter.
- T-1 **Monday**:
  - [ ] Add notes link and video recording link to GitHub discussion thread.
  - [ ] Post description of call, meeting notes, recording, and highlights are posted into #meeting-notes channel.
```

**Comment by joelhans** [comment]
I've been thinking about how to more robustly record community calls. We need something that multiple people know how to function so we're not reliant on a single person being available and thus being a single point of failure.

In addition, if someone is recording their screen, it seems like they wouldn't be able to interact with Discord at all during the call, as that would "pollute" the recording with their personal interactions. Not ideal.

@mateicanavra What have you done in the past to record?

https://craig.chat/home/ looks super easy, as it's just a Discord bot that you invite into the meeting, but it only records audio.

**Reply by jsmorabito** [reply]
an unresolved conflict with recording is consent. you can follow the conversation that was had [here in our discord](https://discord.com/channels/708122962422792194/797888210650398731/829173142155952201). We might want to consider designating certain types of videos/events as ones we'll want to make publicly available which could make the consent process easier. hosting community calls in discord has been very convenient for facilitating serendipitous introductions and interactions, but the lack of an automatic "this call is being recorded" disclaimer before entering the meeting room makes it challenging to also publicly share those community calls without retroactively getting consent from all participants

**Reply by joelhans** [reply]
Thanks for the reminder on the consent part of recording. We should do some more research on what others are doing if they record similar calls.

I saw that Obsidian uses Zoom for this purpose. Not very a very _cool_ tool, and it would fragment our usage in taking one event away from Discord, but Zoom is probably quite ubiquitous at this point among members of the community and has not only the ability to record, but also provide a warning to viewers that recording is starting/ongoing.
